packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -141,7 +141,10 @@ class OptimisticOracleContractMonitor {
           this.oracleType === OptimisticOracleType.OptimisticOracle ? event.proposer : event.request.proposer,
           this.contractProps.networkId
         ) +
-        ` proposed a price for the request made by ${event.requester} at the timestamp ${event.timestamp} for the identifier: ${event.identifier}. ` +
```

Comment:
changed this to be a mark-down link. consider the message below and how this change would have improved it
<img width="575" alt="image" src="https://user-images.githubusercontent.com/12886084/151090770-1257d5a9-7d03-404a-b342-a6a6d58ac9d2.png">